build(docker): Change default provisioner port to 3000

morlinbrot opened the pull request:
Closes #851

On MacOs, port 5000 is used by AirPlay receiver which will make the 'make up' command described in CONTRIBUTING.md fail for Mac users

## Description of change

All occurrences of using port 5000 for the provisioner were changed to use port 3000 instead.

## How Has This Been Tested (if applicable)?
Tested by activating the AirPort receiver (on port 5000) and running 'make up' - works. `docker ps` shows that the provisioner is running on 3000.

oddgrd reviewed (approved):
Thanks, LGTM! :partying_face: We'll also need to change the port here (this is on the latest main version of contributing.md): https://github.com/shuttle-hq/shuttle/blob/d81c201952bfbbeaad9b021166a4a392a1936392/CONTRIBUTING.md?plain=1#L184

morlinbrot commented:
I've rebased my branch on the newest main and added the requested change. I amended the original commit (thought that was cleaner) so there's no new commit to see but the original hash has changed.
